tests/CppUTestExt/MockReturnValueTest.cpp

```diff
@@ -27,6 +27,7 @@
 
 #include "CppUTest/TestHarness.h"
 #include "MockFailureReporterForTest.h"
+#include <climits>
```

Comment:
Direct C includes are not what we want. Shouldn't happen anywhere at the moment, right? Why is this needed?

Reply:
I think we need this for the definition of LLONG_MAX and ULLONG_MAX? Maybe we don't tho?